haiku-apps/commandtimer/commandtimer-2.0~git.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="Launch or repeat commands after a user-chosen count down"
+DESCRIPTION="CommandTimer counts down to zero then executes whatever command \
+is in the top box."
+HOMEPAGE="https://github.com/HaikuArchives/CommandTimer"
+COPYRIGHT="1999 Jason Scaroni"
+LICENSE="Public Domain"
+REVISION="1"
+commit="738de17b4fd70c0d1dcde8118147b43ce8628f4e"
```

Comment:
Great job for on the upstream repo!
Now you just need to change this to:
```
commit="c85fda361d2190688680319fd19d0ee00fb0d4ad"
```
(and remove the Makefile.)

Yep, I know you already knew what to do. I'm just writing this in case someone would like to merge before it's ready ;-)